### PR TITLE
[lcms] Remove duplicate targets

### PR DESCRIPTION
**Pull request by Avgor46**

### CODE REVIEW

File: projects/lcms/build.sh [hunk 1]
```diff
@@ -93,12 +93,12 @@ make -j$(nproc) all
 # build your Sydr targets
```
Comment (apach301) [inline projects/lcms/build.sh:93]: remove comment

File: projects/lcms/build.sh [hunk 2]
```diff
@@ -93,12 +93,12 @@ make -j$(nproc) all
...
 OUT=/lcms_cov
-FUZZERS="cms_link_sydr cmsIT8_load_sydr cms_transform_sydr cms_overwrite_transform_sydr"
+FUZZERS="cms_link cmsIT8_load cms_transform cms_overwrite_transform"
 $CC $CFLAGS /opt/StandaloneFuzzTargetMain.c -c -o main.o
 for F in $FUZZERS; do
     $CC $CFLAGS -c -I /lcms/include \
         /lcms/$F.c -o /lcms/$F.o
     $CXX $CXXFLAGS \
-        main.o /lcms/$F.o -o $OUT/$F \
+        main.o /lcms/$F.o -o $OUT/$F\_sydr \
```
Comment (apach301) [inline projects/lcms/build.sh:102]: Rename to _cov (also fix names in toml configs)

Comment (Avgor46) [reply projects/lcms/build.sh]: done

File: projects/lcms/build.sh (hunk 1 above)
Comment (Avgor46) [reply projects/lcms/build.sh]: done

**Review by apach301 — APPROVED** [review]

**Review by kobrineli — APPROVED** [review]

**Pull request merged**